website/views.py

```diff
@@ -30,6 +30,12 @@ def get_context_data(self, **kwargs):
                     kwargs['partner_entry'] = ordered_entry.entry
             except EntryCollection.DoesNotExist:
                 pass
+
+        # Zip the articles together with the lengths that each preview should be truncated to.
+        # This also takes care of limiting the number of articles to a max of 4.
+        if kwargs['featured_entry_set']:
```

Comment:
hm, I don't think of the template as being inherently more client-editable than the view in a standard django project, but I can see the point. I'll think on that. It would be harder to do on the template, unfortunately, but might be valuable enough to be worth it...